app/views/catalog/_show_mods.html.erb

```diff
@@ -4,7 +4,7 @@
 
 <div class="col-md-12 upper-record-metadata row">
   <%= render 'catalog/record/mods_upper_metadata_section' %>
-  <% if document.druid %>
+  <% if document.druid && (document['dor_resource_count_isi']&.positive? || !document.key?('dor_resource_count_isi')) %>
```

Comment:
Could we make a new method on SolrDocument for `document['dor_resource_count_isi']&.positive? || !document.key?('dor_resource_count_isi')` ? It'd give a good place to explain what we're doing, if nothing else 😁